power/src/power.rs

```diff
@@ -120,14 +124,15 @@ impl TestKind {
         };
         let step_size = 20;
         // There is probably a better way to do this, but it works.
+        // We are rounding up the returned value to the nearest integer.
         for lower in (0..1000).step_by(step_size) {
             let upper = lower + step_size;
             let root = find_root_regula_falsi(lower as f64, upper as f64, f, &mut conv);
             let n = root.unwrap_or(-111.0);
             if n == -111.0 || n.is_nan() {
                 continue;
             }
-            return n.round() as i64;
+            return n.ceil() as i64;
```

Comment:
Why ceil instead of round?

Reply:
Because if sample size needed is `74.x` then we always get `74.0`.
We want to have `75` if `x > 0`.

Reply:
Damn of course 🤦